app/models/api/v1/offering.rb

```diff
@@ -27,9 +30,15 @@ def initialize(student, offering)
       self.endpoint_url = learner ? learner.remote_endpoint_url : nil
       self.total_progress = learner ? learner.report_learner.complete_percent : 0
       self.last_run = learner ? learner.report_learner.last_run : nil
+      self.learner_report_url = learner ? report_portal_learner_url(learner, protocol: protocol, host: host_with_port) : nil
       if learner
         self.detailed_progress = learner.learner_activities.map do |la|
-          { activity: la.activity.name, progress: la.complete_percent }
+          {
+              activity_id: la.activity.id,
+              activity_name: la.activity.name,
```

Comment:
This change from `activity` to `activity_name` could break some API consumers. But it looks like it was just added in the previous offering API changes, so it seems fine to change it.